documentation/gtfs_to_ntfs_specs.md

```diff
@@ -305,13 +305,13 @@ For exemple :
 (2) Depending of the value of the parameter `odt`, the GTFS `timepoint` conversion rules for NTFS `stop_time_precision` are :
 
 * if `odt` is set to `false` or empty:
-  * if `timepoint` is unspecified => `stop_time_precision` equals 0
-  * if `timepoint` equals 1 => `stop_time_precision` equals 0
-  * if `timepoint` equals 0 => `stop_time_precision` equals 1
+  * if `timepoint` is unspecified => `stop_time_precision` equals 0 (Extact)
```

Comment:
`Exact` no?

Reply:
Sorry for the typo, I've fixed it squashing my commit !